### ISSUE TITLE
Video falling way behind audio when halted for a long time

### ISSUE THREAD

**Issue by duckyondiscord** [VueTubeApp/VueTube#419]
### Steps to reproduce

1. Open VueTube
2. Play any video, pause it for a few minutes, resume playing

### Expected behavior

Video starts at the same time as audio and both play in sync

### Actual behavior

Audio starts way before video does resulting in a major desync

### VueTube version

d2934d5

### Android version

Android 12

### Device

OnePlus 6T

### Other details

The issue seems to only occur after the update that added new options in the resolution menu and it seems to be a problem with VueTube not waiting for the video to load before starting to play the audio

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to unstable version **[Latest](https://vuetube.app/install/)**.
- [X] I will fill out all of the requested information in this form.

**Comment by nokanol45** [comment]
Intended behavior for now, the devs are testing adaptive format for videos (i.e. audio and video are separate), which allows VueTube to play videos in higher resolution than 720p30. Referenced in https://github.com/VueTubeApp/VueTube/commit/c14c795ae4418bb59631e12f4655aa44f36f8b2b

**Comment by duckyondiscord** [comment]
> Intended behavior for now, the devs are testing adaptive format for videos (i.e. audio and video are separate), which allows VueTube to play videos in higher resolution than 720p30. Referenced in https://github.com/VueTubeApp/VueTube/commit/c14c795ae4418bb59631e12f4655aa44f36f8b2b

Got it. I assume I should close the issue then, right?

**Comment by nokanol45** [comment]
You can keep it open until the devs fix this issue

**Comment by duckyondiscord** [comment]
> You can keep it open until the devs fix this issue

Okay
